packages/calcite-components/src/components/checkbox/checkbox.tsx

```diff
@@ -55,7 +55,7 @@ export class Checkbox
   @Prop({ reflect: true }) disabled = false;
 
   /**
-   * The ID of the form that will be associated with the component.
+   * The 'ID' of the form that will be associated with the component.
```

Comment:
For cases where the backticks are added to `id`, should the "id" characters be lowercase for consistency with other cases across the doc?

Reply:
I was wondering about that, I chose to leave it capital and see what you thought because there are a couple of cases where there are prop descriptions within the same component referring to an "id". 

So there's two different "ids" being reference and it looks like it was capitalized before to differentiate the "ids" that was referring to the wrapping form of the actual component, while the lowercase version was referring to an "id" on the component itself.

I can def make them lowercase though, WDYT?

Reply:
Okay so I decided to just lowercase the id and wrap it in backticks, I also realized that sentence is present in a lot more components than I previously thought so I made the changes there as well. 

The sentence gets the point across so minimal change to it is needed.